bin/generator.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+command -v java >/dev/null 2>&1 || { echo >&2 "Java executable not found. Aborting."; exit 1; }
+
+DATAHELIX_GENERATOR_HOME=/var/lib/generator
```

Comment:
Same as above

Reply:
fixed this (it should be in /usr/lib, not /var/lib) as per the LFS : https://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/Linux-Filesystem-Hierarchy.html#usr

the generator.sh would be placed (or symlinked) into /usr/bin